source/extensions/clusters/redis/redis_cluster.h

```diff
@@ -282,6 +282,8 @@ class RedisCluster : public Upstream::BaseDynamicClusterImpl {
   const std::string cluster_name_;
   const Common::Redis::ClusterRefreshManagerSharedPtr refresh_manager_;
   const Common::Redis::ClusterRefreshManager::HandlePtr registration_handle_;
+
+  TimeSource& time_source_;
```

Comment:
Why are all these changes needed to the other load balancers? Can this all be removed?

Reply:
yes, good catch